Dockerfile

```diff
@@ -51,23 +51,25 @@ ARG lint_user=lint
 # Install JQ
 RUN curl -sL https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 \
       -o /usr/local/bin/jq                                                                \
-    && chmod +x /usr/local/bin/jq                                                         \
+    && chmod +x /usr/local/bin/jq
 
 ## Install YQ
 RUN curl -sL https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 \
-      -o /usr/local/bin/yq                                                                  \
+      -o /usr/local/bin/yq                                                           \
     && chmod +x /usr/local/bin/yq
 
 ## Install Goss
-    && curl -sL https://github.com/aelsabbahy/goss/releases/download/${GOSS_VERSION}/goss-linux-amd64 \
-         -o /usr/local/bin/goss                                                                       \
-    && chmod +rx /usr/local/bin/goss                                                                  \
+RUN curl -sL https://github.com/aelsabbahy/goss/releases/download/${GOSS_VERSION}/goss-linux-amd64 \
+         -o /usr/local/bin/goss                                                                    \
+    && chmod +rx /usr/local/bin/goss
+
 # Install Hadolint
-    && curl -sL https://github.com/hadolint/hadolint/releases/download/${HADOLINT_VERSION}/hadolint-Linux-x86_64 \
-        -o /usr/local/bin/hadolint                                                                               \
-    && chmod +x /usr/local/bin/hadolint                                                                          \
+RUN curl -sL https://github.com/hadolint/hadolint/releases/download/${HADOLINT_VERSION}/hadolint-Linux-x86_64 \
+        -o /usr/local/bin/hadolint                                                                            \
+    && chmod +x /usr/local/bin/hadolint
+
 # Setup non-root lint user
-    && useradd -ms /bin/bash ${lint_user} \
+RUN useradd -ms /bin/bash ${lint_user} \
```

Comment:
Is there a reason you split this into 2 RUN commands?  Each RUN adds a new layer when building in creasing disk space usage and slowing down the build a little.

Reply:
Merged back into one command